docs/cow-protocol/concepts/order-types/pay-debt-flash-loans.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 6
+---
+
+# Repay debt with collateral using flash loans
+
+A key use case of flash loans is the ability to repay debt with collateral, since flash loans allow users to close or reduce their debt positions without needing upfront liquidity.
+
+Instead of requiring users to have assets readily available to repay their loans, flash loans enable them to temporarily borrow the needed funds, use those funds to repay their debt, and immediately reclaim their locked collateral. Once the collateral is received, it can be swapped or liquidated to cover the borrowed amount, ensuring the loan is repaid within the same transaction.
+
+This approach eliminates the need for users to preemptively sell assets or find external liquidity, making it a highly flexible and efficient way to manage debt positions.
+
+### How to take out a flash loan
+
+This can be achieved using a buy order where:
+
+- The **buy token** is the flash loaned asset.
+- The **buy amount** is the exact amount needed to repay the debt (it must be sufficient to complete the transaction; otherwise, the transaction will revert!).
```

Comment:
```suggestion
- The **buy amount** is the exact amount needed to repay the flash loan (it must be sufficient to complete the transaction; otherwise, the transaction will revert!).
```